clients/src/main/resources/common/message/ApiVersionsResponse.json

```diff
@@ -30,8 +30,8 @@
   "validVersions": "0-3",
   "flexibleVersions": "3+",
   "fields": [
-    { "name": "ErrorCode", "type": "int16", "versions": "0+",
-      "about": "The top-level error code." },
+    { "name": "Error", "type": "error", "versions": "0+",
```

Comment:
We should probably keep ‘ErrorCode’ as this was the name before the auto-generated protocol.